src/transform.js

```diff
@@ -647,31 +646,84 @@ function untransformObject(schema, className, mongoObject, isNestedObject = fals
 
```

Comment:
why don't we do something like
if (isNestedObject) {
   let transformableKeys = ["_id", ...];
   Object.keys(mongoObject).forEach((key) => {
     restObject[key] = untransformObject(schema, className, mongoObject[key], true);
  });
  return restObject;
}